packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1,5 +1,11 @@
 {
     "types": [
+        {
+            "name": "ec2ConnectionType",
+            "type": "string",
+            "allowedValues": ["remoteDesktop", "ssh", "scp", "ssm", "remoteWorkspace"],
+            "description": "Ways to connect to an EC2 Instance"
```

Comment:
Add to https://github.com/aws/aws-toolkit-common/blob/baeeebcb52f45b2bf93ad9f3752efcbeeb6e83eb/telemetry/definitions/commonDefinitions.json#L1118-L1125 ?

P.S. seems like this could be a general `remoteConnectType` metric, but too late :/

Reply:
Oh, like a single metric for all connections with like some metadata for the type (i.e. ec2, ecs, codecatalyst, etc.)?

Also based on https://github.com/aws/aws-toolkit-vscode/blob/master/docs/telemetry.md#development, should I wait for EC2 release to make commons change or no reason to wait?

Reply:
> Also based on https://github.com/aws/aws-toolkit-vscode/blob/master/docs/telemetry.md#development, should I wait for EC2 release to make commons change or no reason to wait?

Mostly a question of whether the metrics are still in development. I think these metrics are fairly settled by now.

Reply:
can remove this now